server/@types/launchpad/index.d.ts

```diff
@@ -1,3 +1,25 @@
+export type RawPrisonerEvent = {
```

Comment:
You can probably import the types from the Swagger. You can use `npx` like BAPV shows https://github.com/ministryofjustice/book-a-prison-visit-staff-ui#imported-types and https://prison-api-dev.prison.service.justice.gov.uk/swagger-ui/index.html#/ and it'll auto-generate a file for you to include.

Then you can make easier to use types like this https://github.com/ministryofjustice/hmpps-developer-portal/blob/main/server/data/strapiApiTypes.ts

Saves you trying to keep track of events from other APIs.

Reply:
Have updated the code to reflect this.